xarray/tests/test_plot.py

```diff
@@ -1208,3 +1213,25 @@ def test_default_labels(self):
         # Top row should be labeled
         for label, ax in zip(self.darray.coords['col'].values, g.axes[0, :]):
             self.assertTrue(substring_in_axes(label, ax))
+
+
+@requires_netCDF4
+class TestDatetimePlot(PlotTestCase):
+
+    def setUp(self):
+        '''
+        Create a DataArray with a time-axis that contains netCDF4.datetime
+        objects.
+        '''
+        month = np.arange(1, 13, 1)
+        data = np.sin(2 * np.pi * month / 12.0)
+
+        darray = DataArray(data, dims=['time'])
+        darray.coords['time'] = \
```

Comment:
style: avoid `\` in favor of implicit line continuation with parentheses: https://www.python.org/dev/peps/pep-0008/#maximum-line-length